posthog/async_migrations/test/test_0006_persons_and_groups_on_events_backfill.py

```diff
@@ -446,3 +465,112 @@ def test_team_id_filter_event_in_team(self):
             },
             events[0],
         )
+
+    def test_postcheck_e2e(self):
+        create_event(event_uuid=uuid1, team=self.team, distinct_id="1", event="$pageview")
+        create_person(
+            team_id=self.team.pk,
+            version=0,
+            uuid=str(uuid1),
+            properties={"personprop": 2},
+            timestamp="2022-01-02T00:00:00Z",
+        )
+        create_person_distinct_id(self.team.pk, "1", str(uuid1))
+
+        self.assertTrue(run_migration())
+
+    def test_check_person_data_success(self):
```

Comment:
let's do this and all next test just based on data without running the migration - so it's easier to debug later when tests are failing & it's faster too.